stompServer.js

```diff
@@ -1,46 +1,59 @@
-var stomp = require('./lib/stomp');
-var StompUtils = require('./lib/stomp-utils');
-var http = require("http");
+var http            = require('http');
 var WebSocketServer = require('ws').Server;
-var EventEmitter = require('events');
-var util = require('util');
+var EventEmitter    = require('events');
+var util            = require('util');
+
+var Stomp           = require('./lib/stomp');
+var StompUtils      = require('./lib/stomp-utils');
+var Bytes           = require('./lib/bytes');
+
+const VERSION = require('./package.json').version;
+
 
 /**
  * STOMP Server configuration
+ *
  * @typedef {object} ServerConfig
  * @param {http.Server} server Http server reference
  * @param {function} [debug=function(args) {}] Debug function
- * @param {string} [path=/stomp] Websocket path
- * */
+ * @param {string} [path=/stomp] WebSocket path
+ * @param {string} [serverName=STOMP-JS/VERSION] Name of STOMP server
+ * @param {number} [heartbeat=[10000, 10000]] Heart-beat; read documentation to config according to your desire
+ */
 
 
 /**
  * @class
  * @augments EventEmitter
- * Create stomp server with config
+ * 
+ * Create Stomp server with config
+ * 
  * @param {ServerConfig} config Configuration for STOMP server
  */
 var StompServer = function (config) {
   EventEmitter.call(this);
-  if (config === undefined) {
+
+  if (config == null) {
```

Comment:
The default config should be used when no `config` parameter is passed to the constructor, shouldn't it? This means `undefined` is ok here

Reply:
Yes. But `config == null` will be true in two cases: if `config` is `undefined` and if is `null`. It's old trick that for example CoffeeScript uses to check if variable exists. This is why i prefer it more then for example more proper: `typeof config == 'undefined'`.
https://stackoverflow.com/questions/5113374/javascript-check-if-variable-exists-is-defined-initialized

Reply:
Ok, thanks for the clarification. I work mostly with linters, which does not allow this, due to the eqeqeq rule. The type conversion is a confusing thing.

Reply:
So should I revert those?